### PR TITLE
Ability to change color of outline using inline styles

### PR DESCRIPTION
**Pull request by devncodex**
## Overview
`tailwindcss-question-mark` is my most used Tailwind plugin so far. Very simple to use and makes debugging a little easier. The only thing I was missing was the option to quickly distinguish between different elements (e.g. for positioning divs, etc). I would find myself adding `bg-[color]-500` to different elements and change the color to make it easier for myself to visualize my changes. 

## Changes

- **createUtilitiesForColor Function**:  If a semantic color name is provided, the function appends the color name to the default question mark (e.g., ?-blue).  This should eliminate the need for using a color picker/dropper tool for precise values. 

## Demo

example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,18 @@
 # tailwindcss-question-mark
 
+> This is a forked version of the
+original [`tailwindcss-question-mark`](https://github.com/GavinJoyce/tailwindcss-question-mark) plugin.
+> See [Usage](#usage) for more info.
+
+- - - -
+
```
Comment (GavinJoyce) [inline README.md]: let's remove this?

File: src/index.js [hunk 2]
```diff
@@ -12,48 +12,78 @@ module.exports = plugin.withOptions(
...
     animationDuration = "0.6s",
     enableAnimation = true,
-    highlightColorStart = "#f16bc9",
-    highlightColorEnd = "#f71fb6",
-    widthStart = "8px",
-    widthEnd = "12px",
+    defaultHighlightColorStart = "#f16bc9",
+    defaultHighlightColorEnd = "#f71fb6",
+    widthStart = "2px",
+    widthEnd = "4px",
```
Comment (GavinJoyce) [inline src/index.js]: how come you're changing the width?

File: src/index.js [hunk 3]
```diff
@@ -12,48 +12,78 @@ module.exports = plugin.withOptions(
...
   } = {}) => {
-    return function ({ addUtilities, e }) {
+    /**
+     * This fork introduces the addition of the `createUtilitiesForColor` function, which dynamically generates utilities colors specified in the tailwind.config.
+     * The function uses the `theme` object to parse the plain-language color names used in tailwind.
+     * Minor changes were made to the `widthStart` and `widthEnd` default values to distinguish between elements that are close in proximity to each other.
+     */
+    return function ({ addUtilities, e, theme }) {
```
Comment (GavinJoyce) [inline src/index.js:20]: let's remove the comment?

**Comment by GavinJoyce** [comment]
This looks great @devncodex, thanks.

I added a few small comments

File: src/index.js (hunk 2 above)
Comment (devncodex) [reply src/index.js]: reverted back to default 8/12 

originally reduced the width to allow some of the outlines to be visible if the elements were positioned too close together, but i cannot seem to recreate the same issue so its reverted. :-)

File: README.md (hunk 1 above)
Comment (devncodex) [reply README.md]: removed

File: src/index.js (hunk 3 above)
Comment (devncodex) [reply src/index.js:20]: removed

**Review by devncodex — COMMENTED** [review]
Removed all comments and reverted the default width start/end

**Comment by GavinJoyce** [comment]
Thanks

**Comment by GavinJoyce** [comment]
https://github.com/GavinJoyce/tailwindcss-question-mark/releases/tag/0.5.0

**Comment by GavinJoyce** [comment]
tailwind-q-colors

**Pull request merged**